README.md

```diff
@@ -206,7 +206,7 @@ This toolbox benefit a lot from open source research and we would like to thank
 
 ## Contributions and code of conduct
 
-Every contribution is welcome and should respect the [contribution guidelines](.github/CONTRIBUTING.md). Each member of the project is expected to follow the [code of conduct](.github/CODE_OF_CONDUCT.md).
+Every contribution is welcome and should respect the [contribution guidelines](contributing.html). Each member of the project is expected to follow the [code of conduct](code_of_conduct.html).
```

Comment:
force a link to the documentation with full url or it will fail on the radme on github:

`https://pythonot.github.io/code_of_conduct.html` and such (will not work until realsease but it will happen shortly)